### PR TITLE
Release

### PR DESCRIPTION
**Pull request by briskt**
- use separate credentials between stg and prod (#3)
- clean up expired access tokens (#5)
- run weekly (#6)

### CODE REVIEW

File: cron/maintenance/serverless.yml [hunk 1]
```diff
@@ -35,4 +35,4 @@ functions:
   cron:
     handler: bin/maintenance
     events:
-      - schedule: cron(30 * * * ? *)
+      - schedule: cron(0 4 * * SUN *)
```
Comment (Baggerone) [inline cron/maintenance/serverless.yml]: Sorry, I should have checked this - it looks like you need a `?` for the day of the month.
```One of the day-of-month or day-of-week values must be a question mark (?).```

Comment (Baggerone) [reply cron/maintenance/serverless.yml]: https://docs.aws.amazon.com/lambda/latest/dg/tutorial-scheduled-events-schedule-expressions.html

Comment (briskt) [reply cron/maintenance/serverless.yml]: Thanks for finding the problem. I remember reading that but somehow convinced myself it wasn't needed here.

**Review by Baggerone — APPROVED** [review]

**Review by fillup — APPROVED** [review]
This could be optimized a bit to scale more in the future by passing the job type as part of the payload to the function and schedule each type of job separately to be two lambdas each triggering a different job on their own schedules.

**Comment by briskt** [comment]
> This could be optimized a bit to scale more in the future by passing the job type as part of the payload to the function and schedule each type of job separately to be two lambdas each triggering a different job on their own schedules.

Yeah, it's a bit on the quick-and-dirty side right now.

**Pull request merged**